web/src/main/java/com/epam/esm/configuration/MyConfig.java

```diff
@@ -0,0 +1,69 @@
+package com.epam.esm.configuration;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.env.Environment;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.orm.hibernate5.HibernateTransactionManager;
+import org.springframework.orm.hibernate5.LocalSessionFactoryBean;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.sql.DataSource;
+import java.util.Properties;
+
+@Configuration
+@ComponentScan("com.epam.esm")
+@EnableTransactionManagement
+
+public class MyConfig {
```

Comment:
Don't name classes with "My.." 😄
Rename it to ControllerConfig or smth else

Reply:
Mede . Rename it to ConnectionConfig.